docker/entrypoint.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+PROMPT_OPTS=()
+BOOTSTRAP_OPTS=()
+BOOTSTRAP=""
+MAIN_NET=""
```

Comment:
I feel this file is overly complex. 

 Also some of the vars are not yet used, eg `MAIN_NET`.